pulser-core/pulser/backend/qpu.py

```diff
@@ -42,7 +42,9 @@ def __init__(
         super().__init__(sequence, connection, mimic_qpu=True)
 
     def run(
-        self, job_params: list[JobParams] | None = None, wait: bool = False
+        self,
+        job_params: list[JobParams] | None = None,
+        wait: bool = False,
```

Comment:
nit: Given there's no actual change here anymore, maybe we can revert the formatting modification?